java/src/org/openqa/selenium/grid/distributor/local/LocalDistributor.java

```diff
@@ -459,6 +459,12 @@ public void remove(NodeId nodeId) {
     Lock writeLock = lock.writeLock();
     writeLock.lock();
     try {
+      Node node = nodes.get(nodeId);
+
+      if (node instanceof RemoteNode) {
+        ((RemoteNode) node).close();
```

Comment:
Could this throw an exception of any kind?

Reply:
I think we are catching any exceptions there in the close() methods itself.

Reply:
👍